.github/release-drafter.yml

```diff
@@ -9,15 +9,16 @@ categories:
   - title: '🚀 Features'
     labels:
       - 'feat'
+      - 'feature'
       - 'enhancement'
-      - 'styles'
   - title: '🐛 Bug Fixes'
     labels:
       - 'fix'
       - 'bug'
   - title: '🧰 Maintenance'
     labels:
       - 'chore'
+      - 'styles'
```

Comment:
we can have a tag for "ui" - which is short and related to styles and looks

Reply:
ok